java/docs/OfferTerms.md

```diff
@@ -0,0 +1,46 @@
+
+
+# OfferTerms
+
+Terms of the offer.
+
+## Properties
```

Comment:
is this a union of all fields for all products?

i think that's not ideal, since we modeled it as a discriminated union

looks like java 17 has sealed classes. is there a way to use that? otherwise this interface is really hard to use

Reply:
Interesting, I thought I ignored these files. We should not put these in the repository. Will remove.

Reply:
Removed.

Reply:
is there an issue with the types though? i'd like to see how the offer term types are being used

Reply:
Yeah, fixed and do type check using instances of: https://github.com/Kanmon/sdk-test/blob/5c2f5358fc5e9db83eb7c3e2d182d75e7ee2a892/kanmon-java-test/src/test/java/com/kanmon/sdk/test/OffersApiTest.java